test/sales_engine_test.rb

```diff
@@ -91,9 +90,46 @@ def test_it_passes_parsed_item_data_to_item_repository
     assert_equal '31163', engine.item_repository.items[6].unit_price
   end
 
+  def test_it_has_a_customer_repository
+    # skip
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
+    engine        = SalesEngine.new(data_location)
+
+    engine.startup
+
+    assert engine.customer_repository.is_a?(CustomerRepository)
+  end
+
+  def test_it_has_a_customer_parser
+    data_location = {customer: "./data/fixtures/customers.csv"}
+    engine        = SalesEngine.new(data_location)
+
+    assert engine.customer_parser.is_a?(CustomerParser)
+    assert_equal "./data/fixtures/customers.csv", engine.customer_parser.data_location
+  end
+
+  def test_default_location_for_customer_data
+    engine   = SalesEngine.new
+    expected = "./data/customers.csv"
+
+    engine.startup
+
+    assert_equal expected, engine.customer_data_location
+  end
+
+  def test_it_passes_parsed_customer_data_to_customer_repository
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [141/80]<br>Space inside { missing.<br>Space inside } missing.